core/trino-spi/src/main/java/io/trino/spi/block/AbstractRowBlock.java

```diff
@@ -72,7 +72,7 @@ public String getEncodingName()
     }
 
     @Override
-    public final Block copyPositions(int[] positions, int offset, int length)
```

Comment:
nit: separate commit

Reply:
IMO it's easier to see the reason for this change if it's part of the commit that overrides this method.

Reply:
I would remove this change for now as it causes test failure and create an issue in Github that SPI checker shouldn't fail for such case